src/site/content/en/progressive-web-apps/promote-install/index.md

```diff
@@ -0,0 +1,382 @@
+---
+layout: post
+title: Patterns for promoting PWA installation
+authors:
+  - pjmclachlan
+date: 2019-06-04
+updated: 2020-02-18
+description: |
+  How to promote installation of Progressive Web Apps and best practices.
+tags:
+  - post
+draft: true
+---
+
+Progressive Web Apps (PWAs) are a pattern for creating app-like, instant
+loading, reliable, and installable websites and why would you want a user to
+install your web app? The same reason you'd want a user to install your native
+app from any app store.
+
+Users who install are your most engaged users. Users who install a PWA
+have better engagement metrics than casual visitors, including more repeat
+visits, longer time on site and higher conversion rates, often at parity with
+native app users on mobile devices.
```

Comment:
Citations would make this paragraph much more compelling

Reply:
@b1tr0t - do you have any citations for this that we can link to?